### ISSUE TITLE
Cannot delete studies in master

### ISSUE THREAD

**Issue by KZzizzle** [ITISFoundation/osparc-issues#263]
## Long story short
I cannot seem to delete studies anymore on master

## Expected behaviour
clicking on "..." and delete, then yes to popup will delete my study

## Actual behaviour
Popup appears, I click yes but the study does not delete for CC rabbit

## Steps to reproduce
![osparc1](https://user-images.githubusercontent.com/28002886/86468184-2af87b80-bd37-11ea-9d42-6beb3484375c.gif)

Note: your environment was attached but will not be displayed
<!--
## Your environment
```json
[
  {
    "name": "osparc-simcore",
    "version": "681fc9c",
    "url": "https://github.com/ITISFoundation/osparc-simcore/tree/681fc9c/"
  },
  {
    "name": "osparc-simcore UI [clean]",
    "version": "681fc9c",
    "url": "https://github.com/ITISFoundation/osparc-simcore/tree/681fc9c/services/web/client/"
  },
  {
    "name": "qooxdoo-compiler",
    "version": "1.0.0-beta.20200111-0015",
    "url": "https://github.com/qooxdoo/qooxdoo-compiler"
  },
  {
    "name": "osparc",
    "version": "1.0.0",
    "url": ""
  },
  {
    "name": "qooxdoo framework",
    "version": "6.0.0-beta-20200110-0942",
    "url": "http://qooxdoo.org"
  },
  {
    "name": "qx-iconfont-fontawesome5",
    "version": "0.2.0",
    "url": "https://github.com/ITISFoundation/qx-iconfont-fontawesome5"
  },
  {
    "name": "qx-iconfont-material",
    "version": "0.1.7",
    "url": "https://github.com/ITISFoundation/qx-osparc-theme"
  },
  {
    "name": "qx-osparc-theme",
    "version": "0.5.4",
    "url": "https://github.com/ITISFoundation/qx-osparc-theme"
  },
  {
    "name": "socket.io",
    "version": "2.2.0",
    "url": "https://github.com/socketio/socket.io"
  },
  {
    "name": "jsondiffpatch",
    "version": "0.3.11",
    "url": "https://github.com/benjamine/jsondiffpatch"
  },
  {
    "name": "jsonTreeViewer",
    "version": "0.6.0",
    "url": "https://github.com/summerstyle/jsonTreeViewer"
  },
  {
    "name": "DOMPurify",
    "version": "2.0.0",
    "url": "https://github.com/cure53/DOMPurify"
  },
  {
    "name": "svg.js",
    "version": "2.7.1",
    "url": "https://github.com/svgdotjs/svg.js"
  }
]
```

**Comment by odeimaiz** [comment]
Somehow you end up having those studies shared with Everyone that you can' delete. That's actually the bug.

**Comment by odeimaiz** [comment]
Fixed in ITISFoundation/osparc-simcore#1634
